cristin-person/src/test/java/no/unit/nva/cristin/person/create/CreateCristinPersonHandlerTest.java

```diff
@@ -326,14 +327,15 @@ void shouldAllowEmptyListOfEmploymentsWhenUserActingAsThemselvesTryToCreateTheir
 
     @Test
     void shouldLogClientSpecificIdentifiersWhenDoingAuthorizedRequests() throws IOException {
-        final var standardOut = System.out;
-        final var outputStreamCaptor = new ByteArrayOutputStream();
-        System.setOut(new PrintStream(outputStreamCaptor));
-        var response = sendQueryWhileMockingIdentifiersUsedForLogging(dummyPerson());
-        System.setOut(standardOut);
-
-        assertThat(outputStreamCaptor.toString(), Matchers.containsString(LOG_MESSAGE_FOR_IDENTIFIERS));
-        assertEquals(HTTP_CREATED, response.getStatusCode());
+        try (var outputStreamCaptor = new ByteArrayOutputStream()) {
+            try (var printStream = new PrintStream(outputStreamCaptor)) {
+                System.setOut(printStream);
+                var response = sendQueryWhileMockingIdentifiersUsedForLogging(dummyPerson());
+                assertEquals(HTTP_CREATED, response.getStatusCode());
+                System.setOut(System.out);
```

Comment:
But System.out is now printstream because you assigned that with setOut?

Reply:
old code was 
```
final var standardOut = System.out;
   System.setOut(standardOut);
```
Unless there are something fancy happening by assigning it a variable first, I cannot see the usecase.